### ISSUE TITLE
Library Usage chapter still suggests `connect_to_device`

### ISSUE THREAD

**Issue by bittner** [Yubico/yubikey-manager#532]
- YubiKey Manager (ykman) version: 5.0.0
- How was it installed?: `pip install yubikey-manager` (latest from PyPI)
- Operating system and version: Ubuntu Linux 18.04.6 LTS Bionic (and higher)
- YubiKey model and version: YubiKey 5 NFC (5.4.3)
- Bug description summary: Code snippets in documentation are outdated

The [Library Usage chapter](https://github.com/Yubico/yubikey-manager/blob/main/doc/Library_Usage.adoc) of the documentation still suggests `connect_to_device`, which was removed/replaced with commit 87bc2e2a0a8e4285725155b3de3feb29942f183f. The documentation should be updated / adapted accordingly.

Steps to reproduce
---

- Try to run the code snippets in [doc/Library_Usage](https://github.com/Yubico/yubikey-manager/blob/main/doc/Library_Usage.adoc).

Expected result
---

- Examples should work.

Actual results and logs
---

- Examples fail with `ImportError` / `NameError`:

```python
...
NameError: name 'connect_to_device' is not defined
...
ImportError: cannot import name 'connect_to_device' from 'ykman.device'
```

**Comment by dainnilsson** [comment]
Thanks for pointing this out! It has now been addressed in [f00c507](https://github.com/Yubico/yubikey-manager/commit/f00c5078967aee452bff873fea72b2c4c4780d77)

**Comment by bittner** [comment]
Thank you for the fix! 💯
